packages/react-jsx-highstock/src/components/RangeSelector/RangeSelector.js

```diff
@@ -1,8 +1,85 @@
-import React from 'react';
-import RangeSelectorInner from './RangeSelectorInner'
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { attempt } from 'lodash-es';
+import { useHighcharts, useAxis, useChart, useModifiedProps } from 'react-jsx-highcharts';
 
-const RangeSelector = props => (
-  <RangeSelectorInner {...props} axisId="xAxis" />
-)
+const RangeSelectorInner = ({ enabled = true, children, ...restProps }) => {
+  const props = { enabled, ...restProps };
+  const [rendered, setRendered] = useState(false);
+  const Highcharts = useHighcharts();
+  const chart = useChart();
+  const axis = useAxis('xAxis');
 
-export default RangeSelector;
+
+  useEffect(() => {
+    if (!axis) return;
+
+    // Workaround inferred from http://jsfiddle.net/x40me94t/2/
+    const chartObj = chart.object;
+    chartObj.options.rangeSelector.enabled = true;
+    Highcharts.fireEvent(chartObj, 'init'); // Pre Highcharts 6.1
```

Comment:
We can get rid of this, as v4 will only support > Highcharts 7.2

Reply:
Can you make that change? I don't really understand what is happening there :)

Reply:
Will do, just about to publish a new alpha too. Then I think we'll merge this to master, we can fix any new issues raised on there.